.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_BACKEND_URL=https://peer-server-stevens.herokuapp.com
```

Comment:
didn't mean to commit this so i'm removing it, but luckily this value isn't sensitive, so we don't need to worry about removing the historical file